pages/common/fzf.md

```diff
@@ -7,6 +7,10 @@
 
 `find {{path/to/search}} -type f | fzf`
 
+- Start finder on all files tracked by git:
```

Comment:
I don't see the usefulness of this command. This does not show a new feature of fzf. And looks very similar to the first and third examples.

Reply:
It's very useful.  While similar to other commands, many users might be unaware that they can search within their git tracked files.

Reply:
I mean can't they make that conclusion from the 1st and 3rd command ? It seems natural to me to pipe the output of any list of files to fzf and search within it. How does this specific command help it further ?

Reply:
Fair enough.

Reply:
Thanks @ryanolsonx. Your contributions are really appreciated though ! Please don't let this discourage you from sending more changes.

Reply:
Thanks